src/pages/solutions/chains.tsx

```diff
@@ -29,193 +28,266 @@ const SolutionsChains: ThirdwebNextPage = () => {
-              shouldShowNoBorder
+
+            <LandingCardWithImage
+              title="Smart Contracts"
+              description="Deploy, read, & write to any smart contract across any EVM-compatible blockchain — and build with thirdweb's audited smart contracts."
+              image={require("../../../public/assets/landingpage/smart-contract-audits-desktop.png")}
+              mobileImage={require("../../../public/assets/landingpage/smart-contract-audits-mobile.png")}
+              TRACKING_CATEGORY={TRACKING_CATEGORY}
+              href="/explore"
+              cardBg="#131418"
+            />
+
+            <LandingCardWithImage
+              title="Payments"
+              description="Point-of-sale tools for fiat & crypto payments — with onramp, swapping, & bridging."
+              image={require("../../../public/assets/landingpage/transaction-fee-v2.png")}
+              mobileImage={require("../../../public/assets/landingpage/transaction-fee-v2.png")}
+              TRACKING_CATEGORY={TRACKING_CATEGORY}
+              href="/sponsored-transactions"
+              cardBg="#131418"
+              colSpan={1}
             />
-          </Card>
-          <Card p={8}>
-            <LandingIconSectionItem
-              icon={require("../../../public/assets/solutions-pages/icons/support.svg")}
-              title="Expert Support"
-              description="Give developers access to best-in-class learning resources and support from professional web3 developers."
-              shouldShowNoBorder
+            <LandingCardWithImage
+              title="Infrastructure"
+              description="Built-in infrastructure so you don't have to worry about RPCs, storage, bundlers or paymasters — or bring your own providers."
+              image={require("../../../public/assets/landingpage/infastructure-desktop.png")}
+              mobileImage={require("../../../public/assets/landingpage/infastructure-mobile.png")}
+              TRACKING_CATEGORY={TRACKING_CATEGORY}
+              href="/rpc-edge"
+              cardBg="#131418"
+              direction="horizontal"
             />
-          </Card>
-          <Card p={8}>
-            <LandingIconSectionItem
-              icon={require("../../../public/assets/solutions-pages/icons/worldclass.svg")}
-              title="Innovation, Built-In"
-              description="Don't let developers fall behind: Give them first access to new standards, infrastructure, and tools before anyone else."
-              shouldShowNoBorder
+            <LandingCardWithImage
+              title="Account Abstraction"
+              description="A best-in-class SDK, gasless transactions, smart account customizability, and managed infrastructure with bundler & paymaster."
+              image={require("../../../public/assets/landingpage/transaction-fee-v2.png")}
+              mobileImage={require("../../../public/assets/landingpage/transaction-fee-v2.png")}
```

Comment:
these are the same, if that is the desired behavior lets make one of them optional in the component and inherit the other one in the case it is not set instead of double-requiring the same asser

Reply:
Ah yeah we didn't have mobile images we should always include them. Included it.